mealie/core/settings/db_providers.py

```diff
@@ -39,22 +39,27 @@ class PostgresProvider(AbstractDBProvider, BaseSettings):
     POSTGRES_SERVER: str = "postgres"
     POSTGRES_PORT: str = "5432"
     POSTGRES_DB: str = "mealie"
+    POSTGRES_URL: str = ""
```

Comment:
I'd suggest that we rename this to POSTGRES_URL_OVERRIDE and nix the re-assigning of POSTGRES_URL in the property. POSTGRES_URL_OVERRIDE is a little more clear in that you're bypassing any validation on inputs. 

It's also weird for a property to set a value that's not just used as a cache IMO and it's only called once or twice so there's no real performance gain to be had here.

Reply:
Alright, I will take another look at this shortly.